README.md

```diff
@@ -1,31 +1,36 @@
-3. Make a Pull Request (PR) for your branch.
-4. Rename your file and directory with the number of the PR and push as a new commits.
-5. Post a link to your PR in #govuk-developers on Slack and to the [govuk-tech-members][govuk-tech-members] Google Group. @mention teams in the PR description if you want particular attention from them.
+3. Make a Pull Request (PR) for your branch and open as a draft.
+4. Rename your file and directory with the number of the PR and push changes.
+5. When your RFC is ready to be commented on, mark you PR as "Ready for review" and set a deadline for comments (2 weeks is a common choice) and record that in the PR description.
+5. Post a link to your PR in #govuk-developers on Slack and to the [govuk-tech-members][govuk-tech-members] Google Group.
 6. GOV.UK members discuss your proposal using both inline comments against your RFC document and the general PR comments section. Non-technical staff will need to create a free Github account in order to comment.
 7. As changes are requested and agreed in comments, make the changes in your RFC and push them as new commits.
-8. Stay active in the discussion and encourage and remind other relevant people to participate. If you’re unsure who should be involved in a discussion, ask your Tech Lead or a Lead Developer. If you start an RFC it’s up to you to push it through the process and engage people.
-9. Once consensus is reached and approvals given using the Github review system, the PR can be merged.
-10. When an RFC is accepted, ensure the GOV.UK Technical community is made aware of it via Slack and the Google Group.
-11. An RFC can be rejected. This can happen if a consensus isn’t reached, or people agree rejecting it is the right thing to do. In this case the PR should be closed with a suitable comment.
+8. Stay active in the discussion and encourage other relevant people to participate. If you’re unsure who should be involved in a discussion, ask your Tech Lead or a Lead Developer. If you start an RFC it’s up to you to push it through the process and engage people.
+9. Once the deadline is reached you are able to merge if the PR has been approved by a member of GOV.UK Senior Tech (if they haven't or you're not sure, you can contact them on Slack with @govuk-senior-tech-people) - this action is the accepting of an RFC.
 
-## Managing Standards
+### If consensus isn't reached
 
-Standards RFCs shouldn’t be substantially altered after they are accepted, although it’s fine to correct typos and other mistakes via a new PR. In order to change a Standard, the original RFC must be superseded by a new one. The process for this is:
+Should consensus not be reached by the deadline, an RFC can either have the deadline extended or have the RFC closed as rejected. For a deadline extension, you should make an effort to inform #govuk-developers about this status and request extra input. If the RFC is being closed as rejected, leave a comment explaining the status and close the PR.
```

Comment:
I wonder if we should have a softer status than rejected for things where we didn't conclusively reject the RFC, but didn't reach consensus.

I can imagine an RFC proposing something sensible but with controversial details getting "rejected", and then later pointed at to say "we agreed we wouldn't do this", when really its status means "we didn't agree that we would do this", which is a lot softer.

Maybe `no_consensus`?

I guess if we add that subtlety we'd also need to agree how to actually close them - close with a comment stating whether it's rejected, or just that consensus wasn't reached and future discussion / RFCs in the same area are welcome?

Reply:
Sure, makes sense - do agree that rejected feels a bit wrong. I'll try adjust the wording as I don't think an RFC that isn't accepted necessarily needs a formal status since it's not merged. 2 tics.

Reply:
I've made an edit, what do you think? https://github.com/alphagov/govuk-rfcs/compare/2dff53506e13771a5a60072f1b84149c426d67a6..9cd187633fc3b2b2bdbc103818fbb0f3c450f687

Reply:
Yeah, lgtm - thanks for that!